pyeudiw/tests/x509/test_x509.py

```diff
@@ -90,29 +93,36 @@ def gen_chain() -> list[bytes]:
         .subject_name(
             x509.Name(
                 [
-                    x509.NameAttribute(NameOID.COMMON_NAME, "leaf.example.org"),
+                    x509.NameAttribute(NameOID.COMMON_NAME, leaf_cn),
```

Comment:
leaf.example.org is a dns name (hostname) while CN can be a URI

please align parameters names with proper values